packages/manager/apps/restricted/src/hooks/useMinimalRights/constants.ts

```diff
@@ -1,9 +1,11 @@
-type RIGHT = {
-  urn: string;
-  subsidiariesExcluded?: string[];
-};
+// type RIGHT = {
+//   urn: string;
+//   subsidiariesExcluded?: string[];
+// };
 
-export const MINIMAL_RIGHTS: RIGHT[] = [
+// To rework
```

Comment:
Can you add `TODO` instead. This will help in identifying the TODOs with other tools like VS Code extension and so on.
```suggestion
// TODO
```

Reply:
👍